source/_data/plugins.yml

```diff
@@ -2277,3 +2277,9 @@
     - options
     - parameters
     - customization
+- name: hexo-fun-time
+  description: A plugin for hexo. Insert fun time into posts.
```

Comment:
Suggest "A tag plugin to insert random Mandarin jokes in a post"

Reply:
The author can use one tag `{% funTime %}` to insert one joke. 
Maybe generating joke according to header (h1, h2, ...) in future, but each part has just one joke.